hazelcast/proxy/topic.py

```diff
@@ -70,6 +71,22 @@ def publish(self, message: MessageType) -> Future[None]:
         request = topic_publish_codec.encode_request(self.name, message_data)
         return self._invoke(request)
 
+    def publish_all(self, messages: typing.Sequence[MessageType]) -> Future[None]:
+        """Publishes a list of messages to all subscribers of this topic
+
+        Args:
+            messages: The message list to be published
+        """
+        try:
```

Comment:
Java does null checks for the parameter and every item inside the collection

Reply:
Added check for None at https://github.com/hazelcast/hazelcast-python-client/pull/570/commits/871f9b1d5c123e864d34f119a74c16f994301a73.